app/controllers/passengers_controller.rb

```diff
@@ -6,10 +6,10 @@ def new
   end
 
   def index
-    @permanent = params[:filter] == ‘permanent’
```

Comment:
Yup. Those are `&lsquo;` and `&rsquo;` ([U+2018][left] and [U+2019][right]). @jforgue ran into this a week or so ago when he was trying to set his MySQL password. Commonly, this comes from copy-and-pasting from someone's blog. Lots of blogging software transforms `'` into `‘` and `’` for *prose* because they look nicer, but code bloggers may be unaware of that fact.

[left]:  http://www.fileformat.info/info/unicode/char/2018/index.htm
[right]: http://www.fileformat.info/info/unicode/char/2019/index.htm